### PR TITLE
Fixed wazuh.yml paths changing /optimize to /data

### PR DESCRIPTION
**Pull request by CPAlejandro**
## Description
This PR change the path of wazuh.yml are modified by changing the /optimize folder for the /data folder

### CODE REVIEW

File: source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-elastic-stack.rst [hunk 1]
```diff
@@ -234,7 +234,7 @@ Upgrading Kibana
                 .. code-block:: console
 
                   # mkdir -p /usr/share/kibana/data/wazuh/config/
-                  # cp /usr/share/kibana/optimize/wazuh/config/wazuh.yml /usr/share/kibana/data/wazuh/config/wazuh.yml
```
Comment (s-ocando) [inline source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-elastic-stack.rst:237]: This path should not be changed. In this step, the old configuration file is copied to the new location during the upgrade process.

File: source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-open-distro.rst [hunk 2]
```diff
@@ -292,7 +292,7 @@ Upgrading Kibana
                 .. code-block:: console
 
                   # mkdir -p /usr/share/kibana/data/wazuh/config/
-                  # cp /usr/share/kibana/optimize/wazuh/config/wazuh.yml /usr/share/kibana/data/wazuh/config/wazuh.yml
```
Comment (s-ocando) [inline source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-open-distro.rst:295]: This path should not be changed. In this step, the old configuration file is copied to the new location during the upgrade process.

**Review by s-ocando — CHANGES_REQUESTED** [review]
Hello @CPAlejandro! Please review the comments below.

File: source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-open-distro.rst (hunk 2 above)
Comment (CPAlejandro) [reply source/upgrade-guide/elasticsearch-kibana-filebeat/upgrading-open-distro.rst]: Indeed, as you mention both changes were wrong. I have already modified them again. Thanks!

**Review by s-ocando — APPROVED** [review]
LGTM

**Pull request merged**